docs/react-testing-library/setup.mdx

```diff
@@ -79,6 +79,7 @@ export { customRender as render }
 
 ```tsx title="test-utils.tsx"
 import React, { FC, ReactElement } from 'react'
+import { Omit } from 'react-redux'
```

Comment:
I'm not sure about the `Omit` here.. I think that the options type should be different as it's the options for our `render` function.